taxonomies/old/containers.txt

```diff
@@ -107,45 +112,52 @@ bg:Корк
 de:Kork
 el:Φελλός
 fr:Bouchon
-nl:Dop
+nl:Dop, kurk
 pt:Cortiça
 hu:Dugó
 wikidata:en:Q49444
 
 fr:Puisette
 hu:Fa
 
+
```

Comment:
I my opinion this blank line can be removed, it is generating a diff in the code difficulting review for a thing that does not changed.

Reply:
@danizavtz FYI, in github when you review PRs you can set not to see white space changes
![Capture d’écran du 2022-09-06 16-18-29](https://user-images.githubusercontent.com/144788/188659241-cbaed551-2014-4cf2-a1b3-5c933e46dcef.png)

Reply:
Thanks, I'll do it next review.